requirements.txt

```diff
@@ -1,7 +1,6 @@
-RadeonOpenCompute/rocm-cmake@04f694df2a8dc9d7e35fa4dee4ba5fa407ec04f8 --build
 sqlite3@3.17 -DCMAKE_POSITION_INDEPENDENT_CODE=On
 boost@1.79 -DCMAKE_POSITION_INDEPENDENT_CODE=On --build
 half,https://github.com/pfultz2/half/archive/1.12.0.tar.gz -X header -H sha256:0a08660b68abb176ebc2a0cdf8de46e3182a7f46c66443bb80dbfaaec98cf969 --build
-ROCmSoftwarePlatform/llvm-project-mlir@rocm-5.3.0 -DBUILD_FAT_LIBMLIRMIOPEN=1
+ROCmSoftwarePlatform/rocMLIR@release/rocm-5.4  -DBUILD_FAT_LIBROCKCOMPILER=1
```

Comment:
@zhanglx13  and @jerryyin

Reply:
You shouldn't use branches just tags or commit hashes, as you can run into reproducibility issues.